pkg/apis/hiveinternal/v1alpha1/clustersync_types.go

```diff
@@ -0,0 +1,121 @@
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	hivev1 "github.com/openshift/hive/pkg/apis/hive/v1"
+)
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ClusterSync is the status of all of the SelectorSyncSets and SyncSets that apply to a ClusterDeployment.
+// +k8s:openapi-gen=true
+// +kubebuilder:subresource:status
+// +kubebuilder:resource:path=clustersyncs,scope=Namespaced
```

Comment:
I generally lean towards not having short names because it is too easy to have conflicts. I am not strictly opposed to them, though, if it is a resource that is used often enough where the short name provides enough value.

Reply:
Not a show-stopper. I do see SRE using `oc get clustersync` regularly (if not super frequently). `csync` saves us 6c. Meh.